### ISSUE TITLE
Existant ava browser runner

### ISSUE THREAD

**Issue by 1000i100** [arlac77/browser-ava#192]
Is your work based on this ?
https://github.com/avajs/ava/pull/3086/files

**Comment by arlac77** [comment]
No, but it is only a simple proof of concept with minimal functionality.

**Comment by 1000i100** [comment]
are you planning to do more ?
I have some issues & features request with my first try.

**Comment by 1000i100** [comment]
If you plan to push it in npm for more use than proof of concept, i would be happy to contribute.

**Comment by arlac77** [comment]
Sounds great, I have no problem publishing the module to npm

**Comment by 1000i100** [comment]
Who ! You work on it so hard ! It's marvelous !

**Comment by 1000i100** [comment]
we can, but we also can ask https://github.com/avajs to add it for extra visibility
